.github/workflows/comment.yml

```diff
@@ -0,0 +1,62 @@
+name: Comment on clippy
+
+on:
+  workflow_run:
+    workflows: ["Rust"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # We lock our failing tests on a known good version and
+        # informationally check the latest as well.  We should
+        # regularly update the known good once we know that tests
+        # pass on it
+        # This needs to stay in sync with rust.yml
+        rust-toolchain: [ 1.66, stable, nightly ]
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: 'Download artifact'
+        uses: actions/github-script@v3.1.0
```

Comment:
I doubt we want to pin on either that specific or old of a version?  If you copied this from the docs/example, it's probably just old and out of date.  Can you check what the latest github-script version is and use that?  (Same with upload-artifact in the other file)